src/game/component/TransformComponent.h

```diff
@@ -25,9 +25,9 @@ class TransformComponent : public IComponent
 			  const glm::vec3& scale = glm::vec3(1,1,1));
 	void Refresh(){}
 	const glm::mat4& GetWorldMatrix() const;
-	const glm::mat4& GetCameraMatrix() const;
 
-	inline void MarkDirty(){ m_PosMatrix[3][0] = 1.f; }
+	inline void MarkDirty(){ m_PosMatrix[0][3] = 1.f; }
```

Comment:
The line exists in both pictures, my change had nothing to do with it. Look slightly to the left on the bottom picture (the texture is scrolling, so the relative position to the middle of the screen doesn't mean much).

With respect to the change to MarkDirty, the position was transposed because the DrawSystem code was modified to no longer transpose matrices when sending them to the GPU, as GLM already produces transform matrices in the expected transposition.
While m_PosMatrix[0][3] is unused by the transformation matrix, and thus could be used as a dirty flag, m_PosMatrix[3][0] acts as a translation factor in the z-direction and would therefore not work for that purpose.